### ISSUE TITLE
Help wanted: Automatically map prefill from altinn2 to altinn3

### ISSUE THREAD

**Issue by ivarne** [Altinn/altinn2-convert#8]
Altinn2 and Altinn3 seems to offer very similar prefill cababilities, but with some differences in naming. A mapping between the different names would be a great small project for someone who wants to contribute.

The available prefills in A3 are documented in a json schema
https://altinncdn.no/schemas/json/prefill/prefill.schema.v1.json

I don't know how to get a list from altinn2, but it looks like you can get something from TUL

Currently I have a draft implementation in https://github.com/Altinn/altinn2-convert/blob/main/Helpers/PrefillConverter.cs, but just a table with matching keys for a2 and a3 would be very usefull.

**Comment by altinnadmin** [comment]
Yeah, we also have this issue https://github.com/Altinn/altinn-studio/issues/4784

**Comment by ivarne** [comment]
Sign me up for the list of "Altinn Studio users are complaining that the documentation on prefill is somewhat lacking" 😉
